app/controllers/api/v1/customers_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::CustomersController < ApplicationController
+
+  def index
+    render json: Customer.all, only: [:id, :first_name, :last_name]
```

Comment:
i'd suggest using a serializer here (i'm assuming y'all just haven't gotten to it yet so no worries!)